standardfiles/cookbook/Dangerfile

```diff
@@ -36,6 +36,11 @@ if !git.modified_files.include?('CHANGELOG.md') && code_changes?
   failure 'Please include a CHANGELOG entry.'
 end
 
+# Require Major Minor Patch version labels
+unless github.pr_labels.grep /minor|major|patch/i
+  failure 'Please add a release label to this pull request'
```

Comment:
this should only apply to code changes and documentation, test changes and build systems should not require a release imo.

Reply:
I agree, however I don't see how it hurts doing a release on every change to the repo. 

This way, you don't end up with unrelated changes in the same release. Plus who doesn't like continuous deployment?

Reply:
Even though this is merged, I am thinking of having it as Warning, simply so it does not show up as test failures on everyone's first builds (Such as forks where they do not have the ability to add the tag)

You also will want to add the checking of the metadata being changed etc